test/scenarios/periodics/el94-src@auto-recovery-rpm.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-liveimg.ks.template "" true
+    launch_vm --boot_blueprint rhel-9.4-microshift-source-isolated
```

Comment:
What's the reason for using non-default blueprint here?

Reply:
Because it's not ostree - it's regular RPM